### PR TITLE
Fix systemtest for edit,delete and purge address

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Bugfix
- Refactoring

### Description
Fix system test for  edit, delete adn purge addtress

### CODE REVIEW

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java [hunk 1]
```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
     }
 
     public WebElement getEditMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Edit')]"));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:139]: if you use id in element, use `.findElement(By.id("some_id"))`

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java [hunk 2]
```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
...
+        return getActionDropDown().findElement(By.xpath("//*[@id='edit-address']/div"));
     }
 
     public WebElement getDeleteMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Delete')]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='delete-address']/div"));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:143]: if you use id in element, use .findElement(By.id("some_id"))

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java [hunk 3]
```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
...
     }
 
     public WebElement getPurgeMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Purge')])]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='purge-address']/div"));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:147]: if you use id in element, use .findElement(By.id("some_id"))

Comment (drcoolsanjeev) [reply systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:147]: @kornys Okay. But in this case, the element is a div and doesn't contain id, so I think xpath is required. 
Here is the element -
<li role="menuitem" id="edit-address">
   <div tabindex="-1" data-key="0" class="pf-c-dropdown__menu-item">Edit</div>
</li>

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java (hunk 3 above)
Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:147]: You can select li element which has id, whatever element you need.

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=shared.standard.web.FirefoxConsoleTest#testFilterAddressesByName

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=shared.standard.web.FirefoxConsoleTest#testFilterAddressesByName

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

**Pull request closed without merging**